src/Caching/SqlServer/src/DatabaseOperations.cs

```diff
@@ -49,9 +52,12 @@ internal class DatabaseOperations : IDatabaseOperations
 
         protected ISystemClock SystemClock { get; }
 
+        protected SqlConnection CreateSqlConnection() => new SqlConnection(ConnectionString)
+            {AccessToken = AccessTokenFunc?.Invoke()};
```

Comment:
Generally we'd use a standard method body for this kind of method. Unless it's a really simple one-liner we tend to stick with standard methods instead of expression bodies.

Reply:
Made the change to adhere to the coding guidelines